clang/lib/Parse/ParseDecl.cpp

```diff
@@ -2890,8 +2890,9 @@ Decl *Parser::ParseDeclarationAfterDeclaratorAndAttributes(
       // ProduceConstructorSignatureHelp only on VarDecls.
       ExpressionStarts = SetPreferredType;
     }
-
-    bool SawError = ParseExpressionList(Exprs, ExpressionStarts);
+    bool HasTrailingComma = false;
+    bool SawError =
+        ParseExpressionList(Exprs, ExpressionStarts, HasTrailingComma);
```

Comment:
This change doesn't seem necessary.

Reply:
@Fznamznon yes, thanks for pointing that out. I've remove it